### PR TITLE
feat: Airswap integration for sAst v3

### PR DESCRIPTION
**Pull request by piersss**
## Description

Added sAST v2 and sAST v3 for airswap module. Problem: sAST v3 tokens does not show up.

## Checklist

- [x] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)

## How to test?

sAST v2:
http://localhost:5001/apps/airswap/balances?addresses[]=0x66ab525022c5291c0c3d7a7199a38b827586f30c&network=ethereum

sAST v3:
http://localhost:5001/apps/airswap/balances?addresses[]=0x2bfbeef6791ce8990d349f30682546583117eee1&network=ethereum

### CODE REVIEW

**Comment by JForsaken** [comment]
@piersss I think you don't have what's latest on master, can you try rebasing when you have the chance? 😄

**Comment by piersss** [comment]
> @piersss I think you don't have what's latest on master, can you try rebasing when you have the chance? 😄

Yeah whoops! It's rebased now.

File: src/apps/airswap/ethereum/airswap.balance-fetcher.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { Network } from '~types/network.interface';
 import { AIRSWAP_DEFINITION } from '../airswap.definition';
 
 const appId = AIRSWAP_DEFINITION.id;
-const groupId = AIRSWAP_DEFINITION.groups.sAST.id;
+const groupId = AIRSWAP_DEFINITION.groups.sASTv2.id;
```
[the remark this answers is not in the document]
Comment (piersss) [reply src/apps/airswap/ethereum/airswap.balance-fetcher.ts]: Ok both token types are in the balance fetcher now, but the v3 tokens still don't show up.

[the remark this answers is not in the document]
Comment (immasandwich) [reply src/apps/airswap/ethereum/airswap.balance-fetcher.ts]: Almost, try this:

```ts
const [sASTv2Balances, sASTv3Balances] = await Promise.all([
  this.getAirswapTokenBalances(address, sASTv2GroupId),
  this.getAirswapTokenBalances(address, sASTv3GroupId),
]);

return presentBalanceFetcherResponse([
  {
    label: 'Staking',
    assets: [...sASTv2Balances, ...sASTv3Balances],
  },
]);
```

**Review by immasandwich — APPROVED** [review]

**Comment by SamIam-0x** [comment]
Hello @piersss !

Looks like you did not provide an address when submitting your PR. I recognize it may have seemed kind of random that we were asking for a wallet address with a PR submission, but we are actually looking for ways to reward Studio contributors in the future. 

Is there an address you’d want to provide? Could even be an empty address, if you do not have an doxxed address

Let me know! Also, if you’d like to privately provide it, feel free to shoot me a TG https://t.me/SamIAm_0x or DM on the Zapper Discord.

Thanks!
Sam

**Pull request merged**